common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -59,6 +58,11 @@
     }
 
 
+    // ************* ANALYTICS *************
+
+    // Ophan pageview ID and browser ID are needed by Google Analytics, which runs sync at the bottom of the page
+    @InlineJs(ophanConfig().body, "ophanConfig.js")
```

Comment:
Sounds like a sensible rename. But I wanted to keep this PR as minimal as possible, so I've left the names alone for now.